tests/pipelines/test_pipelines.py

```diff
@@ -655,7 +655,7 @@ def test_local_save_load_index(self):
                 out = pipe(prompt, num_inference_steps=2, generator=generator, output_type="np").images
 
                 with tempfile.TemporaryDirectory() as tmpdirname:
-                    pipe.save_pretrained(tmpdirname)
+                    pipe.save_pretrained(tmpdirname, variant=variant, safe_serialization=use_safe)
```

Comment:
This should have been serialized with `variant` and `safe_serialization` otherwise the test seems wrong to me.